components/profile/pages/voting-power/network/controller.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect } from "react";
+
+import BigNumber from "bignumber.js";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+
+import Indicator from "components/indicator";
+
+import { useAppState } from "contexts/app-state";
+
+import { useAuthentication } from "x-hooks/use-authentication";
+import useChain from "x-hooks/use-chain";
+
+import VotingPowerNetworkView from "./view";
+
+export default function VotingPowerNetwork() {
+  const { query } = useRouter();
+  const { t } = useTranslation(["common", "profile"]);
+
+  const { state } = useAppState();
+  const { updateWalletBalance } = useAuthentication();
+  const { chain } = useChain();
+
+  const { curatorAddress } = query;
+
+  const oracleToken = {
```

Comment:
this constant can be made into a function that gets `state.Service.network.active` as its argument and returns the formatted object as this is the third time you've declared this exact constant value